Modules/gcmodule.c

```diff
@@ -1021,15 +1042,15 @@ show_stats_each_generations(struct _gc_runtime_state *state)
 
     * The "unreachable" list must be uninitialized (this function calls
       gc_list_init over 'unreachable').
-    
+
 IMPORTANT: This function leaves 'unreachable' with the NEXT_MASK_UNREACHABLE
 flag set but it does not clear it to skip unnecessary iteration. Before the
 flag is cleared (for example, by using 'clear_unreachable_mask' function or
 by a call to 'move_legacy_finalizers'), the 'unreachable' list is not a normal
 list and we can not use most gc_list_* functions for it. */
 static inline void
 deduce_unreachable(PyGC_Head *base, PyGC_Head *unreachable) {
-    validate_list(base, 0);
+    validate_list(base, 0, 0);
```

Comment:
Although I think is clear already, I wonder if we can create a macro for NO flags so these calls look like:

```
validate_list(base, GC_NO_FLAG, GC_NO_FLAG)
```

or similar. I was thinking about this because now that there are two numbers, having two 0's on the call can be confusing at first. Is a very minor thing, so feel free to ignore the suggestion :)

Reply:
I actually find that less clear, but won't object if you do it 😉 .  Clearer still would be, e.g.,
```c
#define VALIDATE_NO_FLAGS(list) validate_list(list, 0, 0)
```
plus 3 more for the other possibilities   But in this file especially, I'm sick to death of having to track down what macros expand to 😉